app/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Auth\LoginRequest;
+use Illuminate\Support\Facades\Auth;
+use Inertia\Inertia;
+
+class LoginController extends Controller
+{
+    public function index()
```

Comment:
Please add return types to all of the methods.

Reply:
I could use some help about returning types. I haven`t used return types declared to a method so far.

Reply:
I have added RedirectResponse type to LoginController -> login method and RegisterController -> store method and it works fine. I can`t find what type should return when using Inertia::render to return inertia view

Reply:
Here's some info on type declarations: https://www.php.net/manual/en/language.types.declarations.php

Reply:
You can check the return types for each Inertia method here: https://github.com/inertiajs/inertia-laravel/blob/master/src/Inertia.php